### ISSUE TITLE
Are jobs with a dependency on 1...n other jobs possible?

### ISSUE THREAD

**Issue by nebularazer** [graphile/worker#458]
### Summary

Is it possible to run a job only if another job has successfully finished?

### Additional context

I have a system which reacts to a users request and i need to:
1. create an entry via an external API -> returns a url
2. create another entry on a different API -> returns a uuid
3. send an email to the user with both the url and the uuid from the two requests

Is it possible to create these jobs at once, so i can see what tasks are still pending for this request while still making sure they are executed properly?

In this example job 1 and 2 can run in parallel and job 3 can only start once both (1,2) are successful.

Thank you!

**Comment by benjie** [comment]
That is out of scope for Worker; you should orchestrate that in your own application logic, even tracking your tasks separately and only performing add_job when it’s time to execute.

Perhaps graphile-saga is of interest? https://github.com/ben-pr-p/graphile-saga
